File: src/main/java/br/com/jiratorio/domain/response/ListIssueResponse.java
package br.com.jiratorio.domain.response;

import br.com.jiratorio.domain.form.IssueForm;
import br.com.jiratorio.domain.sandbox.SandBoxFilter;
import br.com.jiratorio.domain.entity.Board;
import br.com.jiratorio.domain.sandbox.SandBox;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class ListIssueResponse {

    private IssueForm issueForm;

    private Board board;

    private SandBox sandBox;

    private SandBoxFilter sandBoxFilter;

}

File: src/main/kotlin/br/com/jiratorio/domain/response/ListIssueResponse.kt
package br.com.jiratorio.domain.response

import br.com.jiratorio.domain.chart.ChartAggregator
import br.com.jiratorio.domain.entity.embedded.Chart

data class ListIssueResponse(
    val issues: List<IssueResponse>,
    val charts: ChartAggregator,
    val leadTime: Double,
    val weeklyThroughput: Chart<String, Int>
)
